Source/WebCore/css/htmlSwitchControl.css

```diff
@@ -25,5 +25,10 @@
 @namespace "http://www.w3.org/1999/xhtml";
 
 input[type="checkbox"][switch] {
+    appearance: auto;
     margin: initial;
 }
+
+input[type="checkbox"][switch]::thumb, input[type="checkbox"][switch]::track {
+    appearance: auto;
+}
```

Comment:
What are we trying to achieve here? Are we actually planning to expose these UA pseudo-elements to the web for arbitrary styling? I don't think there are many existing non-legacy examples of that (`::file-selector-button`?).

The modern way to style UA shadow is to have scoped style in the shadow tree. But maybe there are performance concerns?

Reply:
Yes, exposing these for arbitrary styling is the eventual goal.